docs/autogenerate/dummy.nix

```diff
@@ -0,0 +1,19 @@
+builtins.mapAttrs (k: _v:
```

Comment:
Why not just map since value is ignored?

Reply:
To produce an attribute per platform

Reply:
slightly better is `lib.genAttrs [ "x86_64-linux" "x86_64-darwin"] f`

Reply:
I'd have to import lib to make that work, which negates cleanup.